examples/video/instream/radiant/pb-ve-radiant.html

```diff
@@ -0,0 +1,329 @@
+
+</code></pre>
+
+
+		<!--body code example-->
+		<div style="width:60vw;">
+			<h4>Place this code in the page body.</h4>
+<pre class="pb-code-hl" style="width:60vw;">
+<!--put body html and javascript here-->
+&lt;script&gt;
+		/* our app where we run player
+			 our app variables */
+		var pbApp = {};
+		pbApp.playerSetup = false;
+		pbApp.prebidTempTag = false;
+		pbApp.debug = true;
+		/* in case pre-bidding takes too long or fails we provide a playerSetupTimeout and fallbackAdTagUrl
+			 to insure player setup happens - this is optional */
+		pbApp.playerSetupTimeout = 5000;
+		pbApp.fallbackAdTagUrl = 'https://www.radiantmediaplayer.com/vast/tags/inline-linear-1.xml';
+		/* no console - no logs */
+		if (typeof window.console === 'undefined' || typeof window.console.log === 'undefined' || typeof window.console.dir === 'undefined') {
+			pbApp.debug = false;
+		}
+
+		/* invokeVideoPlayer may not be defined when bidsBackHandler runs
+			 we pre-defined it here so as to capture the returned adTagUrl to be passed to the player */
+		pbApp.invokeVideoPlayer = function (adTagUrl) {
+			pbApp.prebidTempTag = adTagUrl;
+		};
+
+		/* prebid.js variables */
+		var pbjs;
+		pbjs = pbjs || {};
+		pbjs.que = pbjs.que || [];
+
+		/* Prebid video ad unit
+			 This is a working example but you must use your own settings/bidders for production
+			 More docs at https://prebid.org/prebid-video/video-overview.html */
+		var videoAdUnit = {
+			code: 'video1',
+			mediaTypes: {
+				video: {
+					playerSize: [640, 480],
+					context: 'instream'
+				}
+			},
+			bids: [{
+				bidder: 'appnexus',
+				params: {
+					placementId: 13232361,
+					video: {
+						skippable: true,
+						playback_method: ['auto_play_sound_off']
+					}
+				}
+			}]
+		};
+
+		pbjs.que.push(function () {
+			pbjs.addAdUnits(videoAdUnit);
+
+			pbjs.setConfig({
+				debug: true,
+				cache: {
+					url: 'https://prebid.adnxs.com/pbc/v1/cache'
+				}
+			});
+
+			pbjs.requestBids({
+				bidsBackHandler: function (bids) {
+					if (pbApp.debug) {
+						window.console.dir(bids);
+					}
+					var videoUrl = pbjs.adServers.dfp.buildVideoUrl({
+						adUnit: videoAdUnit,
+						params: {
+							iu: '/19968336/prebid_cache_video_adunit',
+							cust_params: {
+								section: 'blog',
+								anotherKey: 'anotherValue'
+							},
+							output: 'vast'
+						}
+					});
+					pbApp.invokeVideoPlayer(videoUrl);
+				}
+			});
+		});
+
+		/* here we re-define invokeVideoPlayer with Radiant Media Player set-up */
+		pbApp.invokeVideoPlayer = function (adTagUrl) {
+			if (pbApp.playerSetup) {
+				return;
+			}
+			pbApp.playerSetup = true;
+			if (pbApp.debug) {
+				window.console.log('invokeVideoPlayer with Prebid VAST url = ' + adTagUrl);
+			}
+			var src = {
+				mp4: [
+					'https://www.rmp-streaming.com/media/bbb-360p.mp4'
+				]
+			};
+			var settings = {
+				licenseKey: 'Kl8lbz16cz1rN2l2enM9Z2M9NnZvMnllaT9yb201ZGFzaXMzMGRiMEElXyo=',
```

Comment:
Use license key Kl8lZ292K3N6MmVvZD9yb201ZGFzaXMzMGRiMEElXyo= for site prebid.org instead.